Sources/OktaLogger/LoggerCore/OktaLogger.swift

```diff
@@ -116,6 +116,12 @@ public protocol OktaLoggerProtocol {
         - identifier: Identifier of destination which should be removed.
      */
     func removeDestination(withIdentifier identifier: String)
+
+    /**
+     Tracks Analytics events
+     */
+    func trackEvent(name: String, properties: [String: String], logLevel: OktaLoggerLogLevel)
```

Comment:
I guess default implementation does nothing. And even Firebase destination does nothing. I think it is not obvious for developer how to use that API. Is it really should be part of Logger core?